MeshLib/MeshSearch/ElementSearch.cpp

```diff
@@ -10,7 +10,7 @@
 
 #include "ElementSearch.h"
 
-#include <logog/include/logog.hpp>
+#include "BaseLib/Logging.h"
```

Comment:
As far as I can see there isn't any logging output in this file - maybe the include can be removed.